package.json

```diff
@@ -35,6 +36,9 @@
         "tsc"
       ]
     },
+    "build:fast": {
+      "command": "node ./esbuild.mjs"
```

Comment:
Why add it to wire it if it's a simple command with no deps? Is it because of --watch?